src/status_im/ui/screens/accounts/recover/db.cljs

```diff
@@ -1,6 +1,9 @@
 (ns status-im.ui.screens.accounts.recover.db
-  (:require [cljs.spec.alpha :as s]
+  (:require [cljs.spec.alpha :as spec]
             status-im.utils.db))
 
-(s/def ::passphrase :global/not-empty-string)
-(s/def ::password :global/not-empty-string)
\ No newline at end of file
+(defn valid-length? [identity]
+  (> (count identity) 5))
```

Comment:
`identity` probably should be renamed to `password`
Let's move magic value to `status-im.chat.constants` and rewrite condition to `(>= password const/min-password-length)`, change `>` to `>=` because in this case we can give meaningful name for the value.

Reply:
makes sense! will do.

Reply:
oops wrong snippet in previous comment, edited `(>= (count password) const/min-password-length)`

Reply:
wanted to update , but accidentally created new one. https://github.com/status-im/status-react/pull/2421